sigstore-public-deployment-spec.md

```diff
@@ -111,29 +114,33 @@ primary deviations are described in [Spec: Rekor](./rekor-spec.md); however,
 like RFC 6962, the transparency service specification leaves specific parameter
 choices to implementers.
 
-* Base URL: [https://rekor.sigstore.dev/](https://rekor.sigstore.dev/)
-* Hash Algorithm: SHA-256 ([RFC 6234](https://datatracker.ietf.org/doc/rfc6234/); OID 2.16.840.1.101.3.4.2.1)
-* Signature Algorithm:  ECDSA (NIST P-256).
-* Public Key: See [root-signing repo](https://github.com/sigstore/root-signing)
-* Log ID: the SHA-256 hash of the log's public key
-* Maximum Merge Delay: Rekor only returns an index after the merge is complete. Rekor does not support batching. Rekor returns an inclusion proof after waiting for an entry to be included in the log, which is expected to take <1s.
+* Base URL: defined in [Root-of-Trust](#distributing-roots-of-trust)
+* Public Key: defined in [Root-of-Trust](#distributing-roots-of-trust)
+* Merkle Tree Hash Algorithm: SHA-256 ([RFC 6234](https://datatracker.ietf.org/doc/rfc6234/); OID 2.16.840.1.101.3.4.2.1)
+* Signature Algorithm:
+  * Rekor v1: ECDSA (NIST P-256)
+  * Rekor v2: Ed25519
+* Log identity
+  * Rekor v1: Log Id is the SHA-256 hash of the log's public key
+  * Rekor v2: Checkpoint Key Id is defined in [C2SP spec](https://github.com/C2SP/C2SP/blob/main/signed-note.md#signatures)
 * Identity monitoring: [Rekor monitor](https://github.com/sigstore/rekor-monitor)
 
 ### 3.3 Sharding
 
-The Certificate Transparency Log database used by Rekor currently shards every year at a minimum. The sharding is an additive in a sense that old shards will be still available at:  `https://ctfe.sigstore.dev/<SHARD>`. For example, if sharding in 2022, the old shard is available at: [https://ctfe.sigstore.dev/2022/ct/v1/get-sth](https://ctfe.sigstore.dev/2022/ct/v1/get-sth)
+Rekor and Fulcio's Certificate Transparency log currently shard every year at a minimum. Sharding means making the current log instance read-only and creating a new log instance that is writable: this keeps individual log sizes reasonable. The sharding is an additive in a sense that old shards will be still available for reading.
 
-The convention for naming shards is that it will contain the year, followed by the instance. For example, the first shard of the year 2023 should be named 2023 and if other shards are created they will be called 2023-2, 2023-3, and so forth.
+The convention for naming shards is that it will contain the year, followed by the instance. For example, the first shard of the year 2022 should be named 2022 and if other shards are created they will be called 2022-2, 2022-3, and so forth. See e.g. [https://ctfe.sigstore.dev/2022/ct/v1/get-sth](ctfe.sigstore.dev/2022).
```

Comment:
```suggestion
The convention for naming CTFE shards is that it will contain the year, followed by the instance. For example, the first shard of the year 2022 should be named 2022 and if other shards are created they will be called 2022-2, 2022-3, and so forth. See e.g. [https://ctfe.sigstore.dev/2022/ct/v1/get-sth](ctfe.sigstore.dev/2022).
```

Reply:
Sorry, this was my inaccurate correction. Colleen's correct, what's described here is just for the CT log. We should add a line about our naming scheme for Rekor v2 as well.

Reply:
I still don't see how the current text does not apply to rekor v2 URLS just as well, so maybe one of you can provide the correct text?

Reply:
right, I understand that the URLs are formed differently but it seems like
* we are not trying to exactly specify the URL formats here (in fact that seems counterproductive since these are  conventions)
* the current text seems to apply fine to both rekor v2 and fulcio

Is the issue that I'm not providing an example for rekor or what is the ask?

Reply:
Cheers. I simplified text even further, added another example.